ports/libdatachannel/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "libdatachannel",
-  "version-string": "v0.10.1",
+  "version-string": "v0.12.0",
```

```suggestion
  "version-semver": "0.12.0",
```
This field should be `version-semver`. Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions.

---

Hi, should I use version-semver or version ?

JonLiu1993 want to use version.
https://github.com/microsoft/vcpkg/pull/17076#discussion_r607577070

---

Seems you have known the differences between `version `and `version-semver`.

---

This port seems to be the same case as `libjuce`. I checked the version again and thought `version` might be more proper here. Since I cannot make sure if the specific number has meaning. I'm sorry for my mistake, which might make you confused. What do you think?

You can also refer to this doc https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.reference.md
For semver, you can see the details here https://semver.org/

---

To me, right now the version follows both version and version semver rules, so using either is fine.
```
Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.
```
Its true as of now so semver it fine.

---

I guess semver is the right way to do: https://github.com/paullouisageneau/libdatachannel/issues/397#issuecomment-816829786